bin/lib/zopen-md2man

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+# Generate man1/xxx.1.gz from xxx.md
+# Probably throw-away since we are going to use help2man instead
+ME=$(basename $0)
+
+printSyntax()
+{
+  echo "zopen-md2man will generate man pages from markdown (to be removed)" >&2
+  echo "Syntax: zopen-md2man" >&2
+}
+
+if [[ $# -eq 1 ]]; then
+  if [ "x$1" = "x--help" ]; then
+    printSyntax
+    exit 0
+  elif [ "x$1" = "x--version" ]; then
+    zopen --version $ME
+    exit 0
+  else
+    echo "Unrecognized option $1. Processing terminated" >&2
+    printSyntax
+    exit 8
+  fi
+fi
+
+set -e
+zopen_version='0.6.3' ## add code to perform zopen --version once supported
```

Comment:
Should this be 0.7.0?

Reply:
Yeah. Hm. I changed it to 0.7.0 - maybe it got lost on the cleanup. Will fix